test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -448,6 +448,35 @@
               "No exception thrown even though no file existed for copying")
           (fs/copy copy cacert))))))
 
+(deftest retrieve-ca-crl!-test
+  (testing "CRL file copied when it doesn't already exist"
+    (let [tmp-confdir (fs/copy-dir confdir (ks/temp-dir))
```

Comment:
K, disregard the concern then.  It seems like we would want to allow concurrent testing at some point, but this pattern of initializing shared state at the top of a `deftest` isn't what's holding us back.

For what it's worth, I took a stab at using `[com.holychao/parallel-test "0.3.0"]` and I ended up having to mark the majority of `deftest` variables as :static, and I didn't even finish.  We'd need to refactor most of our tests to execute them concurrently so this probably needs to be addressed at a different level.